core/ui/src/main/java/org/phoebus/ui/spi/ContextMenuEntry.java

```diff
@@ -48,11 +57,48 @@ public default Image getIcon()
      */
     public Class<?> getSupportedType();
 
+    /**
+     * Invoke the action associated with this context menu contribution
+     * @throws Exception on error
+     */
+    public default void call() throws Exception
+    {
+        throw new UnsupportedOperationException(getName() + ".call() Is not implemented.");
```

Comment:
Just looking at the API, i.e. all those default `call` variants which all throw exceptions, I'm not sure:
a) Can I ignore them all, since there's a default for all of them, so the compiler is happy?
b) Do I actually need to implement all, otherwise the Exception will cause me to fail at runtime?
c) Do I just need to implement one, because whatever uses it will fall back to the "other" variant of call until one succeeds?

Maybe all it needs is another comment in the Javadoc of what one needs to do.

Reply:
>  second iteration

OK